src/legacy/server/saved_objects/saved_objects_mixin.js

```diff
@@ -55,7 +55,7 @@ function getImportableAndExportableTypes({ kbnServer, visibleTypes }) {
   );
 }
 
-export async function savedObjectsMixin(kbnServer, server) {
+export function savedObjectsMixin(kbnServer, server) {
```

Comment:
seems unrelated

Reply:
Applying a boy scout rule. It doesn't add a lot of noize.